manifests/service.pp

```diff
@@ -1,12 +1,25 @@
 # configure, enable, and start pulpcore services
 # @api private
 class pulpcore::service {
+
```

Comment:
A future lint (that's on the agenda to add) will complain about this empty line.

Reply:
Bummer, I like the readability the empty line creates.